compiler/src/dotty/tools/dotc/core/NameKinds.scala

```diff
@@ -295,7 +295,7 @@ object NameKinds {
   val DocArtifactName: UniqueNameKind        = new UniqueNameKind("$doc")
   val UniqueInlineName: UniqueNameKind       = new UniqueNameKind("$i")
   val InlineScrutineeName: UniqueNameKind    = new UniqueNameKind("$scrutinee")
-  val InlineBinderName: UniqueNameKind       = new UniqueNameKind("$elem")
+  val InlineBinderName: UniqueNameKind       = new UniqueNameKind("$bind")
```

Comment:
I find `bind` too unspecific, and of the wrong kind. If we use it, it should be `binder`, not `bind`. But maybe `proxy` is better?

Reply:
Indeed, renamed to `proxy`